onnxruntime/python/tools/transformers/onnx_model_phi.py

```diff
@@ -296,11 +298,20 @@ def get_phi2_edge_dict(self) -> dict:
         edge_dict["l_input_ids_"] = "input_ids"
         edge_dict["key_states"] = "past_key_0"
         edge_dict["value_states"] = "past_value_0"
-        for i in range(self.num_hidden_layers):
+        for i in range(1, self.num_hidden_layers, 1):
             edge_dict[f"key_states_{i}"] = f"past_key_{i}"
             edge_dict[f"value_states_{i}"] = f"past_value_{i}"
             edge_dict[f"model_layers_{i}_1"] = f"present_key_{i}"
             edge_dict[f"model_layers_{i}_1_1"] = f"present_value_{i}"
+
```

Comment:
There is a slight graph discrepancy between the dynamo export on cpu and gpu. In Olive, cpu is the default export system.

Reply:
Please change conver_to_onnx.py to force to use cpu for export for consistency.

Reply:
The code handles both scenarios. In convert_to_onnx.py, we prefer to use cuda if it's supported since cpu export is slower.